business/layer.go

```diff
@@ -189,17 +187,22 @@ func NewWithBackends(k8s kubernetes.ClientInterface, prom prometheus.ClientInter
 	temporaryLayer.TLS = TLSService{k8s: k8s, businessLayer: temporaryLayer}
 	temporaryLayer.TokenReview = NewTokenReview(k8s)
 	temporaryLayer.Validations = IstioValidationsService{k8s: k8s, businessLayer: temporaryLayer}
+
+	// TODO: Use client manager or passed in clients rather than hardcoding the home cluster.
+	clusterClients := map[string]kubernetes.ClientInterface{"home": k8s}
```

Comment:
Ideally it wouldn't be `"home"` but come from the istio config that Kiali is deployed alongside of. The other cluster names will come from the remote secret kubeconfig files. I'm unsure if these need to match with their name in their cluster's istio config but they at least need to be unique. At any rate, the `home` cluster is just the cluster that Kiali is deployed in. It's at least helpful while we are iteratively adding multi cluster support and I think it will be useful for some of the business services that gather info about the mesh and the cluster but it may not make sense to keep in the future. I'll add a global const for it.